posthog/models/organization.py

```diff
@@ -31,11 +31,6 @@
 if TYPE_CHECKING:
     from posthog.models import Team, User
 
-try:
-    from ee.models.license import License
-except ImportError:
-    License = None  # type: ignore
```

Comment:
This is causing some gnarly circular imports for me when trying to use `Organization` in the ee `Role` model, happy to take suggestions here if there's a better way about this than importing the `License` only when we need it for this file